config/cmake/addGitHooks.cmake

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8.7)
+
+find_package(Git)
+if(NOT GIT_FOUND)
```

Comment:
This shouldn't be a fatal error, otherwise it may be impossible to build on a system that's using the tar archive for a release on a system that doesn't have git installed. I think wrapping the rest of this file inside `if(GIT_FOUND)` instead would avoid problems. It should probably also include a version check to make sure the version of git used supports setting a hook path. So maybe use a condition like `if(GIT_FOUND AND (GIT_VERSION_STRING VERSION_GREATER "2.9"))`.

Reply:
We need to be able to build HELICS in some configuration with no git.  So we can't have a requirement on Git that can't be bypassed.  If there are some requirements on GIT that can't be bypassed already in place we need to fix them.    No issue using git if it is available but we do need to be able to build without it.